mycli/main.py

```diff
@@ -346,15 +347,15 @@ def prompt_tokens(cli):
                     logger.error("traceback: %r", traceback.format_exc())
                     self.output(str(e), err=True, fg='red')
                     continue
-
-                destroy = confirm_destructive_query(document.text)
-                if destroy is None:
-                    pass  # Query was not destructive. Nothing to do here.
-                elif destroy is True:
-                    self.output('Your call!')
-                else:
-                    self.output('Wise choice!')
-                    continue
+                if self.destructive_warning:
+                    destroy = confirm_destructive_query(document.text)
+                    if destroy is None:
+                        pass  # Query was not destructive. Nothing to do here.
+                    elif destroy is True:
+                        self.output('Your call!')
+                    else:
+                        self.output('Wise choice!')
+                        continue
```

Comment:
No worries. I was pretty confused! I had set my IDE to not use idents and was trying to figure out where and how I inserted tabs.